src/main/java/pro/mikey/autoclicker/AutoClicker.java

```diff
@@ -1,5 +1,6 @@
 package pro.mikey.autoclicker;
 
+import ca.weblite.objc.Client;
```

Comment:
Client?

Reply:
Fixed lol. Tried to type MinecraftClient it ended up importing Client.